src/librustc_unicode/u_str.rs

```diff
@@ -369,7 +369,7 @@ impl<'a> DoubleEndedIterator for Graphemes<'a> {
 }
 
 // https://tools.ietf.org/html/rfc3629
-static UTF8_CHAR_WIDTH: [u8; 256] = [
+const UTF8_CHAR_WIDTH: [u8; 256] = [
```

Comment:
This and the ascii maps below are getting somewhat large, how come they're constants instead of statics?